main.js

```diff
@@ -1338,7 +1340,8 @@ function sendSSH(method, cb) {
         });
 
         // substitute single ' with '"'"' - 'aaa'bbb' -> 'aaa'"'"'bbb' using string concat
-        const pwd= adapter.config.password.replaceAll("'", "\'\"\'\"\'");
+        //const pwd= adapter.config.password.replaceAll("'", "\'\"\'\"\'"); - replaceAll requires node.15
+        const pwd= adapter.config.password.replace(/\'/g, "\'\"\'\"\'");
```

Comment:
Do it really needs to be escaped? I think not

Reply:
Ok, seems u are right - as always :-)
I'll remove the unneeded escapes. Wunder why vscode did not mark them. It did at other adapters.